src/index.html

```diff
@@ -1,15 +1,82 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>HTML Form</title>
   <link rel="stylesheet" href="./style.css">
 </head>
+
 <body>
-  <h1>HTML Form</h1>
+  <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+    <fieldset class="block-field">
+      <legend>Personal information</legend>
+      <label class="block-for-input">Surname:
+        <input type="text" name="surname" autocomplete="off" required>
+      </label><br>
+      <label class="block-for-input">Name:
+        <input type="text" name="name" autocomplete="off" required>
+      </label><br>
+      <label class="block-for-input">How old are you?
+        <input type="number" name="age" value="12" min="7" max="100">
+      </label><br>
+      <label class="block-for-input">Full date off birth:
+        <input type="date" name="birthdate">
+      </label><br>
+      <label class="block-for-input">I accept the term of the agreement
+        <input type="checkbox" name="terms">
+      </label>
+    </fieldset>
+    <fieldset class="block-field">
+      <legend>Registration</legend>
+      <label class="block-for-input">E-mail:
+        <input type="email" name="email" placeholder="email@example.com">
+      </label><br>
+      <label class="block-for-input">Password:
+        <input type="password" name="password" minlength="8" maxlength="64">
+      </label>
+    </fieldset>
+    <fieldset class="block-field">
+      <legend>An interesting fact about you!</legend>
+      <label class="block-for-input">Do you love cats?
+        <input type="radio" name="cats">Yes
+        <input type="radio" name="cats">No
+      </label><br>
+      <label class="block-for-input">What is your favorite color?
+        <input type="color" name="color" value="#ddd">
+      </label><br>
+      <label class="block-for-input">What time do you go to bed?
+        <input type="time" name="time" step="any">
```

Comment:
краще використовуй числа
```suggestion
        <input type="time" name="time" step="1">
```

Reply:
`name="time"` - довлі розмито